### DISCUSSION TITLE
Merry Christmas 🎄

### DISCUSSION THREAD

**Discussion by rex-schilasky** [eclipse-ecal/ecal#149] [General]
Merry Christmas to my fantastic eCAL developer team and to everyone contributing or using eCAL or just reading this 😀.

Stay all healthy!

**Comment by floriangeis** [comment]
Thank you Rex. 
Same to you and your loved ones!  🎄 🎅 🎁

**Comment by rex-schilasky** [comment]
12 MONTH LATER ..

🎄🎄🎁🎁🎄🎄🎄🎄🎄🎅🎄🎄🎁🎄🎅🎄🎄🎄🎁🎄🎄🎄🎅🎄🎄🎁🎄🎄🎄🎄🎄🎄🎅🎄🎄🎄🎄🎄🎅
**Merry Christmas to all eCAL developers, contributors and users. All the best to you and your loved ones**
🎄🎄🎄🎅🎄🎄🎄🎄🎄🎄🎄🎁🎄🎄🎄🎄🎄🎅🎄🎄🎄🎄🎁🎄🎄🎄🎅🎄🎄🎁🎁🎄🎄🎄🎄🎄🎅🎄🎄.

Enjoy the nice time, stay safe and see you all in 2022  :-)
